src/components/Editor.vue

```diff
@@ -726,6 +730,7 @@ export default {
 
 		outlineToggled(visible) {
 			this.emit('outline-toggled', visible)
+			emit('text:outline:toggle', visible)
```

Comment:
I'm wondering if we should scope the event by the file that is being edited, just in case there might be multiple editor instances at the same time?

Otherwise maybe you could also just add a callback handler to the editor api similar to https://github.com/nextcloud/text/blob/a0f9aa64c10200479c6c1643a0ce33de221b5a76/src/editor.js#L57-L59 which triggers the callback on the vue emit